NEXT_CHANGELOG.md

```diff
@@ -373,4 +373,12 @@ By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/p
 
 ## 🛠 Maintenance
 
+## 🚀 Features
+
+### Display licenses.html diff in CI if the check failed ([#1524](https://github.com/apollographql/router/issues/1524))
+
+The CI check that ensures that the `license.html` file is up to date now displays what has changed when the file is out of sync.
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o)
+
```

Comment:
While I appreciate the thoroughness — do we think this might actually be a distraction to our end users who won't know how this impacts them?

Anyhow, happy with it staying, but I don't know that I'd categorize it as a "Feature" — maybe it could live in a ":broom:Housekeeping" section?

```suggestion
## 🚀 Features

### Display licenses.html diff in CI if the check failed ([#1524](https://github.com/apollographql/router/issues/1524))

The CI check that ensures that the `license.html` file is up to date now displays what has changed when the file is out of sync.

By [@o0Ignition0o](https://github.com/o0Ignition0o)

```

Reply:
Oh i wanted to put it in the maintenance section, fixing this ^^'

Reply:
fixed in [0d04af1](https://github.com/apollographql/router/pull/1524/commits/0d04af15548047ed2c59ef817db429d76800f896)